### ISSUE TITLE
[BUG] Mesh Merging results in lighting / reflection problems

### ISSUE THREAD

**Issue by DrBlackRat** [euan142/LazyOptimiser#16]
**Issue:**  
Now after #15 is closed I've discovered a new issue with the mesh merging.  
After the meshes have been merged, I've discovered that there are now a few issues with the reflections and lighting.

**Attachments:**  
The left side is after using Lazy Optimiser and the right is the original:
![image](https://user-images.githubusercontent.com/46327609/230962803-cbecdee3-47e8-41e9-9f29-fdebcabca8e7.png)

Here is the back side, the left is the original and the right is after using Lazy Optimiser:
![image](https://user-images.githubusercontent.com/46327609/230962884-7e8b0c1d-2004-448a-adbd-0cebb2730755.png)

Here are a few other screenshots:
![image](https://user-images.githubusercontent.com/46327609/230963105-4b5a8a8c-250c-4be9-8ff1-204a6f24999c.png)
![image](https://user-images.githubusercontent.com/46327609/230963205-6ad401e0-01c3-496d-90ba-4217d8f2e0b5.png)
